src/SiteInformation/ContactAndLocation/RenderActionModal.js

```diff
@@ -128,6 +132,25 @@ const RenderActionModal = (props) => {
                 </Button>
               </Tooltip>
             </Grid>
+            <Grid item>
```

Comment:
could you reorder the buttons to be Edit Data, Edit Protocols, Reassign Site, Unenroll, Comment, Map